.github/workflows/release.yml

```
@@ -91,9 +103,15 @@
       - name: Build native binary (Unix)
         if: matrix.platform != 'win32'
         run: |
+          # Add architecture-specific flags for cross-compilation
+          ARCH_FLAGS=""
+          if [ "${{ matrix.platform }}" = "linux" ] && [ "${{ matrix.arch }}" = "arm64" ]; then
+            ARCH_FLAGS="-Dquarkus.native.container-build=true -Dquarkus.native.builder-image=graalvm"
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)